scripts/deploy-surge.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+if [ “$TRAVIS_PULL_REQUEST” == “false” ]; then
```

Comment:
You can also test if this is the master branch as not all branches are always deployed.